snafu-derive/src/lib.rs

```diff
@@ -98,6 +99,76 @@ impl Transformation {
     }
 }
 
+/// SyntaxErrors is a convenience wrapper for a list of syntax errors discovered while parsing
+/// something that derives Snafu.  It makes it easier for developers to add and return syntax
+/// errors while walking through the parse tree.
+struct SyntaxErrors {
+    inner: Vec<syn::Error>,
+}
+
+impl SyntaxErrors {
+    fn new() -> SyntaxErrors {
```

Comment:
This is fine, but for what it's worth, I would have just added `#[derive(Default)]` and then called `SyntaxErrors::default()`

Reply:
I'm not sure why I usually forget that `Default` is a thing, but this is an easy improvement I'll make. :)